### DISCUSSION TITLE
Dialog - scroll not working on IOS

### DISCUSSION THREAD

**Discussion by fiskpatte** [tailwindlabs/headlessui#2169] [Help] [answered]
I can't make modals using Dialog scrollable on iPad or iPhone.
It works fine when I view the same page with phone/tablet-dimensions on desktop, including when using Safari.

I have tried a bunch of examples, including the purchased modal component from Tailwind's component library.
Everyone has the exact same issue. 
I wonder if anyone of you have an example where the modal scroll actually works also on these devices.

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your bug report!
Much appreciated! 🙏

Can you open an issue with a minimal reproduction repo?
Also make sure that you are on the latest version.

**Reply by fiskpatte** [reply]
I upgraded to the latest version and everything worked perfectly, rookie mistake not to check that first. Thanks a lot, I love the work you guys do!

**Comment by 1abdurRehman** [comment]
@RobinMalfait  This issue is occurring again with @headlessui/vue@1.7.19, whereas scrolling works fine with @headlessui/vue@1.7.13.
